src/agents/agent.py

```diff
@@ -361,6 +400,14 @@ def execute(self, prompt: str, project_name_from_user: str = None) -> str:
         self.coder.save_code_to_project(code, project_name)
 
         self.agent_state.set_agent_active(project_name, False)
+        if self.project_manager.get_auto_commit(project_name):
+            project_path = self.project_manager.get_project_path(project_name)
+            if self.git == None:
+                self.git = Git(project_path, self.base_model)
+
+            commit_message = self.git.generate_commit_message(project_name, conversation,code_markdown)
```

Comment:
There is no reference of conversation and code_markdown in execute function

Reply:
can u add both these in execute function

Reply:
Sure will add that

Reply:
may i know ur discord username?